source/kra/parser.d

```diff
@@ -72,6 +72,8 @@ KRA parseKRAFile(ZipArchive file)
 
 	importAttributes(kra, image.children[0]);
 
+	pointCloneLayers();
```

Comment:
@LunaTheFoxgirl Is this the right placement for finalization?

For reference, can you point me to an example of 2-step parsing?